spec/dummy/app/models/invoice.rb

```diff
@@ -92,4 +92,18 @@ def self.colors
   def set_default_state
     self.state ||= 'pending'
   end
+
+  def self.ransackable_attributes(_auth_object = nil)
```

Comment:
I understand that now it is mandatory to do this, right?

Reply:
[yep](https://github.com/activerecord-hackery/ransack/pull/1400)